### ISSUE TITLE
could not get source code

### ISSUE THREAD

**Issue by CHETHAN562** [neverbiasu/ComfyUI-SAM2#6]
torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ..\aten\src\ATen\native\TensorShape.cpp:3588.)
final text_encoder_type: bert-base-uncased
UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
UserWarning: torch.utils.checkpoint: the use_reentrant parameter should be passed explicitly. In version 2.4 we will raise an exception if use_reentrant is not passed. use_reentrant=False is recommended, but if you need to preserve the current default behavior, you can pass use_reentrant=True. Refer to docs for more details on the differences between the two variants.
UserWarning: None of the inputs have requires_grad=True. Gradients will be None
!!! Exception during processing!!! could not get source code

**Comment by neverbiasu** [comment]
I cannot find any useful error for debugging.
